WcaOnRails/app/helpers/server_status_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module ServerStatusHelper
+  def panel_classes(missing_empty, outdated_empty)
+    class_panel = class_missing = class_outdated = "success"
+    title = "All good, the translation is up to date!"
+    unless missing_empty && outdated_empty
+      if missing_empty
+        class_panel = class_outdated = "warning"
+        title = "Some items in the translation are outdated"
+      elsif outdated_empty
+        class_panel = "warning"
+        class_missing = "danger"
+        title = "Some items in the translation are missing"
+      else
+        class_panel = class_outdated = "warning"
+        class_missing = "danger"
+        title = "Some items in the translation are missing and some other are outdated"
```

Comment:
Personal preference, but I'd probably have written this to make the 4 cases painfully clear:

```ruby
if missing_empty && outdated_empty
  ...
elsif missing_empty && !outdated_empty
  ...
elsif !missing_empty && outdated_empty
  ...
elsif !missing_empty && !outdated_empty
  ...
else
  fail
```

I acknowledge that a lot of people find this style pretty weird, but I like to show to the reader that I *did* explicitly consider every case, and I didn't forget about a possible state (hence the `else fail` that's impossible to hit) (Don't feel a need to respond to this.)

Lastly, I found the variable names `missing_empty` and `outdated_empty` a bit confusing here (does `missing_empty` mean "this is the empty that we're missing"?), but I cannot think of better names for them. Maybe `translations_are_missing` and `translations_are_outdated`?